In [1]:
import requests
import json

## 1. Create a new user

#### We will create a new user called John:

In [13]:
requests.get('http://localhost:3500/create/user/John')

<Response [200]>

## 2. Assign user to a chat

#### To to it, we will need a chat name,  and then a user name to add into this chat:

In [14]:
requests.get('http://localhost:3500/chat/Sports/user/John')

<Response [200]>

## 3. Add a new message into a chat

#### To add a new message we will have to create the user and the chat first, then we will be able to do it, otherwise it will raise a friendly error:

In [15]:
requests.get('http://localhost:3500/chat/Sports/user/John/message/I support FC Barcelona, they are the best')

<Response [200]>

## 4. Get all message from a chat as a list

#### We will make a list with all message from a specific chat:

In [17]:
requests.get('http://localhost:3500/chat/Sports/list').json()

['The match was awesome, Barça won again',
 'I love Barça and futbol in general, I think all of them are playing really good. You are awesome guys! I love you! Goodnight :)',
 'Messi is shit, real Madrid is much better',
 'I really hate Barça they don´t even know how to play!',
 'They are amazing, Messi is playing so good in this season',
 'I support FC Barcelona, they are the best']

## 5. Analyse the sentiment to the chat

#### To analyse it, we will use all sentence created in the chat to be able to analyse one by one, after that we will know if the sentiment of the chat is negative, neutral or positive. To to that we will use NLTK.

In [18]:
requests.get('http://localhost:3500/chat/Sports/sentiment').json()

{'compound': 0.8316, 'neg': 0.0, 'neu': 0.391, 'pos': 0.609}

### In the chat Sports our exemple, we can figure out it has a 0.8316 from 1 of compound, it means it has an avarage of positives sentence.